src/main/java/pl/craftserve/radiation/LugolsIodinePotion.java

```diff
@@ -309,5 +328,43 @@ public Duration duration() {
         public Optional<String> drinkMessage() {
             return Optional.ofNullable(this.drinkMessage);
         }
+
+        public Recipe getRecipe() {
+            return recipe;
+        }
+
+        public static class Recipe {
+            private final boolean enabled;
+            private final PotionType basePotion;
+            private final Material ingredient;
+
+            public Recipe(boolean enabled, Material ingredient, PotionType basePotion) {
+                this.enabled = enabled;
+                this.ingredient = Objects.requireNonNull(ingredient, "ingredient");
+                this.basePotion = Objects.requireNonNull(basePotion, "basePotion");
+            }
+
+            public Recipe(ConfigurationSection section) {
+                if (section == null) {
+                    section = new MemoryConfiguration();
+                }
+
+                this.enabled = section.getBoolean("enabled", true);
+                this.ingredient = Material.matchMaterial(Objects.requireNonNull(section.getString("ingredient", "GHAST_TEAR")));
+                this.basePotion = PotionType.valueOf(section.getString("base-potion", "THICK"));
```

Comment:
This can throw `IllegalArgumentException`. If it did, throw a new `InvalidConfigurationException`.